vault/logical_system_paths.go

```diff
@@ -64,38 +84,208 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 		{
 			Pattern: "config/ui/headers/$",
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ListOperation: b.handleConfigUIHeadersList,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ListOperation: &framework.PathOperation{
+					Callback: b.handleConfigUIHeadersList,
+					Summary:  "Return a list of configured UI headers.",
+				},
 			},
 
 			HelpDescription: strings.TrimSpace(sysHelp["config/ui/headers"][0]),
 			HelpSynopsis:    strings.TrimSpace(sysHelp["config/ui/headers"][1]),
 		},
 
 		{
-			Pattern:         "generate-root(/attempt)?$",
+			Pattern: "generate-root(/attempt)?$",
+			Fields: map[string]*framework.FieldSchema{
+				"otp": &framework.FieldSchema{
```

Comment:
This field is currently not documented, and maybe on purpose. This makes me wonder if we need an `Unpublished` property for the fields as well. I totally see paths having one or more deprecated fields which we don't ever want to give out by default.

Reply:
Or maybe marking this as `Deprecated` is the solution.

Reply:
It looks like this one was removed from docs in the past month. re: Unpublished fields... maybe? The Unpublished attribute added to methods was primarily to cut down on documentation "noise" where the path is present in the backend structure as an implementation detail but isn't accessed in that way. I'm a little less sure about Unpublished fields. I mostly feel like if a field is present and is used by Vault, it should be documented. We can deprecate the field, or document why no one should use it or that it's "internal" only. 

But I'm a bit torn on this one tbh. I like a clean set of docs, but I think undocumented parameters just beg for more questions about why they're not documented. Thoughts, @jefferai or @briankassouf ?

Reply:
Practically speaking, in cases like this where the path definitions exist solely for documentation purposes, I can easily remove the definition. To achieve the same suppression in normal `framework.Path` structures, we'd need to update FieldSchema (e.g. with an Unpublished attribute). That would not be a big change if we chose that route.

Reply:
Let's mark this as deprecated for now